tests/sources/fixtures/salesforce/fixture.py

```diff
@@ -203,7 +196,7 @@ def describe_sobject(_version, _sobject):
 
 @app.route("/sfc/servlet.shepherd/version/download/<_download_id>", methods=["GET"])
 def download(_download_id):
-    return io.BytesIO(bytes(fake_provider.get_html(), encoding="utf-8"))
+    return io.BytesIO(bytes(fake_provider.get_text(), encoding="utf-8"))
```

Comment:
Question: why the switch to `get_text`?

Reply:
There wasn't any reason for that, i just updated it to add text instead of html content for checking. Do you want me to revert it?

Reply:
Yeah please keep `get_html` so that we're also testing text extraction in functional tests